config.yaml

```diff
@@ -19,6 +19,13 @@ options:
     type: string
     description: Comma separated list of domains to be allowed to federate.
     default: ''
+  ip_range_whitelist:
```

Comment:
Do we need to "whitelist" here? Can we use allowlist instead just to keep the code inclusive?

Reply:
Unfortunately, this is how the configuration is set in Synapse:
https://matrix-org.github.io/synapse/latest/usage/configuration/config_documentation.html#ip_range_whitelist
I'm afraid that changing it here to a different name might be confusing.

Reply:
I'll mark this as resolved but feel free to re-open. I can modify that in this ongoing PR
https://github.com/canonical/synapse-operator/pull/119